ui/mirage/config.js

```diff
@@ -960,6 +976,10 @@ export default function () {
   });
 
   this.get('/var/:id', function ({ variables }, { params }) {
+    let variable = variables.find(params.id);
+    if (!variable) {
+      return new Response(404, {}, {});
+    }
```

Comment:
This emerged because I've started looking up the `nomad/jobs` variable in isolation when loading a given job/variables page now.

Reply:
The HCP repo has helpers for these little common mirage things. They're quite handy; basically turns this into `return notFound()`.